dev/src/Command/SplitCommand.php

```diff
@@ -235,13 +244,15 @@ private function processComponent(
         $parentTagSource,
         $updateReleaseNotes,
         ?Packagist $packagist,
+        ?string $branchToPush = null
     ) {
         $output->writeln('');
         $tagName = 'v' . $component->getPackageVersion();
         $repoName = $component->getRepoName();
         $componentId = $component->getId();
         $isAlreadyTagged = $github->doesTagExist($repoName, $tagName);
         $defaultBranch = $github->getDefaultBranch($repoName) ?: 'main';
+        $branchToPush = $branchToPush ?? $defaultBranch;
```

Comment:
another SUPER nit, but we could make this a little more optimal by only calling `$github->getDefaultBranch` (which is a network call) when we need to (instead of always).

In this case, we will almost always be calling it, but it's a best practice.

```suggestion
        if (is_null($branchToPush)) {
            $defaultBranch = $github->getDefaultBranch($repoName) ?: 'main';
        }
```

Reply:
First option seems fine :D
Done.